cmd/frontend/internal/app/jscontext/jscontext.go

```diff
@@ -378,6 +403,38 @@ func derefString(s *string) string {
 	return *s
 }
 
+func resolverUserPermissions(ctx context.Context, userResolver *graphqlbackend.UserResolver, licenseInfo *hooks.LicenseInfo) *PermissionsConnection {
```

Comment:
should it be like this?
```suggestion
func resolveUserPermissions(ctx context.Context, userResolver *graphqlbackend.UserResolver, licenseInfo *hooks.LicenseInfo) *PermissionsConnection {
```